### DISCUSSION TITLE
【图大，用流量的请勿点开】「雪霁逢椿」

### DISCUSSION THREAD

**Discussion by Myitian** [share121/inter-knot#1578] [其他]
![雪霁逢椿 封面](https://raw.githubusercontent.com/Myitian/-01/main/雪霁逢椿cover.gif)
**【此贴完全加载需要约 500 MiB 流量】**

原视频：[《原神》PV短片——「雪霁逢椿」](https://b23.tv/BV1ZF411g7EZ)

好吧，其实是测试一下在100MiB限制下用GIF放视频，[sws_dither的不同抖动算法](https://ffmpeg.org/ffmpeg-scaler.html)的效果，防止有人不小心加载，内容放在第一个评论

**Comment by Myitian** [comment]
## 无抖动
|属性|值|
|-|-|
|sws_dither|`none`|
|文件大小|101939 KiB|
|分辨率|642x361|

![雪霁逢椿 none](https://raw.githubusercontent.com/Myitian/-01/main/雪霁逢椿@none.gif)
评价：不做抖动，当然就全是色块啦

## Bayer 抖动
|属性|值|
|-|-|
|sws_dither|`bayer`|
|文件大小|101933 KiB|
|分辨率|516x290|

![雪霁逢椿 bayer](https://raw.githubusercontent.com/Myitian/-01/main/雪霁逢椿@bayer.gif)
评价：还行，sws_dither设为自动（`auto`）用的就是这个算法，封面也是这个；但是遇到奇数宽度的画面时，sws_dither会回落至`ed`算法

## 基于加法的算术抖动
|属性|值|
|-|-|
|sws_dither|`a_dither`|
|文件大小|101864 KiB|
|分辨率|505x284|

![雪霁逢椿 a_dither](https://raw.githubusercontent.com/Myitian/-01/main/雪霁逢椿@a_dither.gif)
评价：有一种六角形网格的感觉

## 基于异或的算术抖动
|属性|值|
|-|-|
|sws_dither|`x_dither`|
|文件大小|102366 KiB|
|分辨率|476x268|

![雪霁逢椿 x_dither](https://raw.githubusercontent.com/Myitian/-01/main/雪霁逢椿@x_dither.gif)
评价：比`a_dither`分布更随机一些

## 误差扩散抖动
|属性|值|
|-|-|
|sws_dither|`ed`|
|文件大小|102099 KiB|
|分辨率|303x170|

![雪霁逢椿 ed](https://raw.githubusercontent.com/Myitian/-01/main/雪霁逢椿@ed.gif)
评价：噪点会抖来抖去，导致压缩率下不来

**Comment by Buer-Nahida** [comment]
用Git LFS，还能再提

**Comment by capacitor1** [comment]
🤔文件体积小是拿低画质换来的。。

我不是很建议靠压缩来发，压缩完了啥都看不清了

不过以目前的文件大小限制，除了直接发链接，其他办法不可行（悲

**Comment by VacuolePaoo** [comment]
这还是gif吗
